LoRaEngine/example.env

```diff
@@ -29,8 +29,8 @@ PKT_FWD_SPI_DEV=2
 ##################################
 NET_SRV_VERSION=1.0.7
 NET_SRV_LOG_LEVEL=Error
-NET_SRV_LOGTO_UDP=false
-NET_SRV_LOG_TO_UDP_ADDRESS=127.0.0.1
+NET_SRV_LOG_TO_TCP=false
```

Comment:
everyone needs to update their .env files. This is a breaking change for everyone using the kit today and needs to be release noted.

Reply:
@Mandur Are we tracking release notes somewhere for the next version? Something to add to the [upgrade page](https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/upgrade/) for vNext?

/cc @roel4ez

Reply:
We do not have such feature. that is a good idea thought ;)

Reply:
Tracking in #825.

Reply:
We should definitely prep release notes and keep track of changes we are making.